esphome/components/mitsubishi_itp/climate.py

```diff
@@ -308,8 +335,14 @@ async def to_code(config):
         # Add sensor as source
         SELECTS[CONF_TEMPERATURE_SOURCE_SELECT][2].append("Thermostat")
 
-    # Traits
+    # If RTC defined
+    if CONF_TIME_SOURCE in config:
+        rtc_component = await cg.get_variable(config[CONF_TIME_SOURCE])
+        cg.add(getattr(muart_component, "set_time_source")(rtc_component))
+    elif CONF_UART_THERMOSTAT in config:
+        raise cv.RequiredFieldInvalid(f"{CONF_TIME_SOURCE} is required if {CONF_TS_UART} is set.")
```

Comment:
- The time source should only be required if KEM is on, correct?
- I'm still a little uneasy about always requiring a time source when it's not actually needed for any functionality that we're aware of.  On the other hand, as I said somewhere else, if it's behind the KEM switch for now I'm not as concerned.

Reply:
I'm still *personally* of the opinion that ESPHome made a mistake by not automatically providing a time source as part of their system, so I don't really see it as a problem. KEM will always need it (well, so long as we want accurate times on a linked MHK), and the ESP will need it anyways if you ever wanted to do any sort of on-device scheduling.

Either way, moved behind the emulation mode flag since I did forget this.

Reply:
> so long as we want accurate times on a linked MHK

I mean, the MHK's internal clock isn't *that* inaccurate, so it's more a question of if the timestamps are optional or can be left out somehow.  We don't really have any *need* for the ESP and MHK to have their clocks precisely synced that I'm aware of.

Reply:
When in Kumo mode, you cannot set the time manually on an MHK. It will always take whatever the upstream device decides to give it. Meaning, the time must be set *before* connected to an ESP.

Given that the response packet that sets time does not contain a flag of any sort, I suspect it's set whether we want to handle it or not. I can see if removing the timestamp and sending all zeroes does anything interesting (probably not), but I also honestly don't see a downside to providing an accurate time source to the ESP.